manifests/init.pp

```diff
@@ -83,6 +83,13 @@
     }
   }
 
+  if ($pe_console_url == undef) {
+    $final_console_url = "https://${settings::report_server}"
```

Comment:
I'd mention that the intent here is to default to the MoM without using the PE modules, and `report_server` seems to be the best candidate for that.

(The PE module bit is important for otherwise, we do have https://github.com/puppetlabs/puppet-enterprise-modules/blob/master/modules/puppet_enterprise/manifests/init.pp#L48)

Reply:
Done. See if you like it.